dap_aria_mapping/notebooks/novelty/pipeline_openalex_novelty.py

```diff
@@ -0,0 +1,81 @@
+""" 
+Script/pipeline to calculate novelty scores for OpenAlex papers
+Uses typer to create a command line interface
+
+Usage examples:
+
+All levels, full dataset:
+python dap_aria_mapping/notebooks/novelty/pipeline_openalex_novelty.py
+
+One level, test dataset:
+python dap_aria_mapping/notebooks/novelty/pipeline_openalex_novelty.py --taxonomy-level 1 --test
+"""
+from dap_aria_mapping import logging, PROJECT_DIR
+import dap_aria_mapping.getters.openalex as oa
+import dap_aria_mapping.utils.novelty_utils as nu
+import typer
+
+OUTPUT_FOLDER = PROJECT_DIR / "outputs/interim"
+
+
+def main(
+    taxonomy_level: int = 0,
+    test: bool = False,
+    n_test_sample: int = 1000,
+):
+    """
+    Calculate novelty scores for OpenAlex papers
```

Comment:
It would be quite nice if you briefly describe the method / calculation, for code legacy sake (and not relying on checking the very nice PR message you wrote on it)